app/src/main/res/layout/drag_drop_test_activity.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.recyclerview.widget.RecyclerView xmlns:android="http://schemas.android.com/apk/res/android"
-  xmlns:app="http://schemas.android.com/apk/res-auto"
-  android:id="@+id/drag_drop_recycler_view"
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/drag_drop_test_fragment_placeholder"
   android:layout_width="match_parent"
-  android:layout_height="wrap_content"
-  app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager" />
+  android:layout_height="match_parent"
+  tools:context=".app.story.StoryActivity" />
```

Comment:
Nit:
1. This file is missing an EOF newline.
2. This tools context seems incorrect.

Reply:
Updated.